odxtools/diaglayer.py

```diff
@@ -441,7 +441,7 @@ def get_receive_id(self):
         """CAN ID to which the ECU listens for diagnostic messages"""
         # TODO (?): When using the dict notation to access comparams,
         #           wouldn't it be prettier to allow the dot in the id?
-        com_param = self.communication_parameters["ISO_15765_2__CP_UniqueRespIdTable"]
+        com_param = self.communication_parameters["ISO_15765_2__CP_UniqueRespIdTable_from_ISO_15765_2"]
```

Comment:
it could be possible that the name of the document fragment for `ISO_15765_2.CP_UniqueRespIdTable` is different from `ISO_15765_2`, so it's probably a good idea to filter:

```python
cps = [ cp for cp in self.communication_parameters if cp.short_name.startswith("ISO_15765_2__CP_UniqueRespIdTable_from")
assert len(cps) == 1, "Communication parameter `ISO_15765_2.CP_UniqueRespIdTable` not found"
com_param = cps[0]
```

That said, once #82 has been merged, this needs to be modified once more...

Reply:
Hi @andlaus - This is a blocker for us. We would really appreciate, if you could approve this MR.

Reply:
I see. is there an issue with going with the `.startswith()` approach from above? I really would like to avoid to hard-code the name of the document fragment of the communication parameters...

Reply:
My bad. From your previous comment, I thought you were OK with this MR. I get this error when running the above code snippet. I don't see a short_name attribute for CommunicationParameterRef.

```
Traceback (most recent call last):
  File "test.py", line 6, in <module>
    print("physical_request_id = ", hex(ecu_variant.get_receive_id()))
  File "/Users/smileprem/Code/sibros/odxtools/odxtools/diaglayer.py", line 445, in get_receive_id
    cps = [cp for cp in self.communication_parameters if cp.short_name.startswith('ISO_15765_2__CP_UniqueRespIdTable_from')]
  File "/Users/smileprem/Code/sibros/odxtools/odxtools/diaglayer.py", line 445, in <listcomp>
    cps = [cp for cp in self.communication_parameters if cp.short_name.startswith('ISO_15765_2__CP_UniqueRespIdTable_from')]
AttributeError: 'CommunicationParameterRef' object has no attribute 'short_name'
```

Reply:
sorry that I was a bit unclear: I *prefer* the more comprehensive solution, but if there is a good reason why it won't work, I will (somewhat grudgingly ;)) merge this PR as a quick fix.

> `AttributeError: 'CommunicationParameterRef' object has no attribute 'short_name'`

ok, replace `short_name` by `id_ref`(I have not tested this before making the original suggestion, sorry...)

Reply:
BTW: #82 should also fix this issue...

Reply:
That's awesome. Let's try to merge the other MR and close this one.